packages/tasklist-client-node-js/src/__test__/restclient.spec.ts

```diff
@@ -44,130 +41,99 @@ describe('TasklistRESTClient', () => {
 
     afterAll(() => zbc.close())
 
-    it('can request all tasks', async () => {
-        const tasklist = new TasklistRESTClient()
-        const tasks = await tasklist.getAllTasks()
-        console.log('tasks', tasks)
-        expect(tasks.length).toBeGreaterThan(0)
-    })
-
-    it('can request a task with parameters', async () => {
-        const tasklist = new TasklistRESTClient()
-        const tasks = await tasklist.getTasks({
-            state: TaskState.CREATED,
+    describe('Read operations', () => {
+        it('can request all tasks', async () => {
+            const tasklist = new TasklistRESTClient()
+            const tasks = await tasklist.getAllTasks()
+            expect(tasks.length).toBeGreaterThan(0)
         })
-        expect(tasks.length).toBeGreaterThan(0)
-    })
 
-    it('can request tasks with fields', async () => {
-        const tasklist = new TasklistRESTClient()
-        const tasks = await tasklist.getTasks({
-            state: TaskState.CREATED,
+        it('can request a task with parameters', async () => {
+            const tasklist = new TasklistRESTClient()
+            const tasks = await tasklist.getTasks({
+                state: TaskStateREST.CREATED,
```

Comment:
All specs in this file that call `getTasks` now pass in the `TaskStateREST` enum instead of the original `TaskState`. Further explanation below, where `TaskStateREST` is defined.